js/package-lock.json

```diff
@@ -8,6 +8,9 @@
       "name": "cov-tiles",
       "version": "0.0.1",
       "dependencies": {
+        "@bufbuild/buf": "^1.32.2",
+        "@bufbuild/protobuf": "^1.10.0",
```

Comment:
From reading the docs briefly at https://github.com/bufbuild/protobuf-es it looks like `"@bufbuild/protobuf"` is the runtime library such that I think the other deps can be in `devDependencies` to keep the overall library size down.

Also, overall I think using a fully featured library like `bufbuild` for now seems ideal, and then as a future step (when library size matters) we could consider moving using the tiny `pbf` library.